beacon-chain/blockchain/forkchoice/process_block_test.go

```diff
@@ -483,3 +483,45 @@ func TestSaveInitState_CanSaveDelete(t *testing.T) {
 		t.Errorf("wanted: %d, got: %d", len(store.initSyncState), params.BeaconConfig().SlotsPerEpoch)
 	}
 }
+
+func TestFilterBlockRoots_CanFilter(t *testing.T) {
+	ctx := context.Background()
+	db := testDB.SetupDB(t)
+	defer testDB.TeardownDB(t, db)
+
+	store := NewForkChoiceService(ctx, db)
+	fBlock := &ethpb.BeaconBlock{}
+	fRoot, _ := ssz.SigningRoot(fBlock)
```

Comment:
FYI, this is going to fail when it syncs with v0.9.2 since we deprecated signing root.. Please fix that after merge.